src/style.css

```diff
@@ -1,3 +1,54 @@
+html {
+  font-family:
+    system-ui,
+    -apple-system,
+    BlinkMacSystemFont,
+    'Segoe UI',
+    Roboto,
+    Oxygen,
+    Ubuntu,
+    Cantarell,
+    'Open Sans',
+    'Helvetica Neue',
+    sans-serif;
+}
+
 body {
   margin: 0;
+  justify-content: space-around;
```

Comment:
The `justify-content` property is used for flex or grid containers. To use it on the `body`, you need to set `display: flex;` or `display: grid;` on the `body` element.